contracts/hyllar/src/client.rs

```diff
@@ -9,10 +9,14 @@ pub struct Builder<'b> {
 }
 
 impl HyllarToken {
-    pub fn default_builder<'b>(&self, builder: &'b mut TransactionBuilder) -> Builder<'b> {
-        builder.init_with("hyllar".into(), self.as_digest());
+    pub fn default_builder<'b>(
```

Comment:
I called "default_builder" in other contrats the builder for the builtin contract, so either keep this logic and have a 2nd function "builder" that takes the contract name as parameter, or rename this function into "builder" like in staking